src/Symfony/Component/Validator/Constraints/UrlValidator.php

```diff
@@ -20,8 +20,8 @@
  */
 class UrlValidator extends ConstraintValidator
 {
-    const PATTERN = '~^
-            (%s)://                                 # protocol
+    const PATTERN = '<^
+            (%protocol%)://                         # protocol
```

Comment:
why these two changes? better not do them imho, and continue using sprintf (requires doubling the `%` below)